runtime/src/main/java/org/corfudb/runtime/clients/ClientMsgHandler.java

```diff
@@ -70,19 +70,20 @@ public ClientMsgHandler(IClient client) {
      */
     @SuppressWarnings("unchecked")
     public boolean handle(CorfuMsg message, ChannelHandlerContext ctx) {
```

Comment:
the return value is not used. I suggest removing it and throwing an IllegalStateException if a message type needs to be handled, but there is no mapping to a handler.

Reply:
Not relevant to my change anymore